collections.go

```diff
@@ -17,6 +17,11 @@ func newCollectionsClient(sl *sling.Sling) *CollectionsClient {
 	return &CollectionsClient{sl: sl}
 }
 
+// Structure gives access to id's document structure.
```

Comment:
With the `Collections.Create` and `Documents.Create` we started to add the API reference.
See [this line](https://github.com/ioki-mobility/go-outline/pull/35/files#diff-0ba7b976e3f9b89e648e3ba228386a911e97656963fb62cf41ff46697c0bf79cL29).

Do we want to continue with that? 🤔

Reply:
Well, the objective of our package is to be intuitive enough to does not feel the need to look at the actual HTTP API. Ideally we only mention the API once in say README and that is it. But sure will add it, philosophical things can be polished later.